app.js

```diff
@@ -66,6 +67,17 @@ if (app.get('env') === 'production') {
 app.use(session(session_opts));
 
 //==============================================================================
+// CSRF MIDDLEWARE
+//==============================================================================
+
+app.use(cookieParser());
+
+app.use((err, req, res, next) => {
```

Comment:
mmm, If I remove the err param all the tests start failing. I need to check why.